sonata-project/translation-bundle/2.3/config/packages/sonata_translation.yaml

```diff
@@ -0,0 +1,16 @@
+sonata_admin:
+    assets:
+        extra_stylesheets:
+            - bundles/sonatatranslation/css/sonata-translation.css
+
+sonata_block:
+    blocks:
+        sonata_translation.block.locale_switcher: ~
+
+sonata_translation:
+    locales: ['en']
+    default_locale: 'en'
+    #gedmo:
```

Comment:
Im not very happy with configuration in comments like this? 
I would suggest uncomment them or remove them.

Reply:
I removed commented lines